webpack.config.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const isProduction = process.env.NODE_ENV === "production";
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const extractSass = new ExtractTextPlugin({
+  filename: "dropkick.css"
+});
+
+// I hate this, but I guess you can't pass disable to uglify..
+const plugins = [ extractSass ];
+isProduction ? plugins.push(new UglifyJSPlugin()) : null ;
```

Comment:
It felt more dirty than being able to do: 
```js
new UglifyJsPlugin({
  disable: !isProduction
})
```

But I guess this isn’t terrible.